packages/auth/src/graphql/schema.graphql

```diff
@@ -0,0 +1,76 @@
+type Query {
+  "Fetch a page of grants."
+  grants(
+    "Paginating forwards: the cursor before the the requested page."
+    after: String
+    "Paginating backwards: the cursor after the the requested page."
+    before: String
+    "Paginating forwards: The first **n** elements from the page."
+    first: Int
+    "Paginating backwards: The last **n** elements from the page."
+    last: Int
+  ): GrantsConnection!
```

Comment:
@wilsonianb this would concern the backend package as well, but should the input variables for pagination be pulled out into a separate type?